tests/sub_port_interfaces/templates/ptf_nn_agent.conf.ptf.j2

```diff
@@ -0,0 +1,11 @@
+[program:ptf_nn_agent]
```

Comment:
Will adding a soft link serve better here since we don't need to maintain two copies?

Reply:
Those templates have some differences between input values. Thanks!